azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/AnnotationHandlerImpl.java

```diff
@@ -84,10 +92,25 @@ public FunctionConfiguration generateConfiguration(final Method method) {
 
         patchStorageBinding(method, bindings);
 
+        config.setRetry(getRetryConfigurationFromMethod(method));
         config.setEntryPoint(method.getDeclaringClass().getCanonicalName() + "." + method.getName());
         return config;
     }
 
+    private Retry getRetryConfigurationFromMethod(Method method) throws AzureExecutionException {
+        final FixedDelayRetry fixedDelayRetry = method.getDeclaredAnnotation(FixedDelayRetry.class);
+        final ExponentialBackoffRetry exponentialBackoffRetry = method.getDeclaredAnnotation(ExponentialBackoffRetry.class);
+        if (!Objects.isNull(fixedDelayRetry) && !Objects.isNull(exponentialBackoffRetry)) {
+            throw new AzureExecutionException(MULTI_RETRY_ANNOTATION);
+        } else if (fixedDelayRetry != null) {
+            return Retry.createFixedDelayRetryFromAnnotation(fixedDelayRetry);
+        } else if (exponentialBackoffRetry != null) {
+            return Retry.createExponentialBackoffRetryFromAnnotation(exponentialBackoffRetry);
+        } else {
+            return null;
+        }
```

Comment:
q1: line 103 is conflict with line 109/110.

Reply:
q2: I guess it should be ok even if both fixedDelayRetry and exponentialBackoffRetry are null. right ?

Reply:
103 means exponentialBackoffRetry and fixedDelayRetry could not exists the same time (I'll change to use != to make it more clearly